frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -690,6 +690,26 @@ func (e *cvdCommandTimeoutErr) Error() string {
 	return fmt.Sprintf("cvd execution with args %q timed out", strings.Join(e.args, " "))
 }
 
+type filterBuffer struct {
+	b    *bytes.Buffer
+	sink *bytes.Buffer
+}
+
+func newFilterBuffer(sink *bytes.Buffer) *filterBuffer {
+	return &filterBuffer{b: &bytes.Buffer{}, sink: sink}
+}
+
+func (b *filterBuffer) Write(p []byte) (n int, err error) {
+	if n, err := b.b.Write(p); err != nil {
+		return n, err
+	}
+	return b.sink.Write(p)
+}
```

Comment:
Could this be achieved using [io#MultiWriter](https://pkg.go.dev/io#MultiWriter) instead?

Reply:
we ended up removing the struct but io.MultiWriter was the way to go.